cmd.go

```diff
@@ -60,6 +68,19 @@ func infocmd(opt Options) error {
 		releases = []Release{*release}
 	}
 
+	if err := renderer(tags, releases); err != nil {
+		return err
+	}
+
+	return nil
```

Comment:
If you see the following at the end of a function:

```
if err := something() ; err != nil {
  return err
}
return nil
```

Then it can always be replaced by:

```
return something()
```